x-pack/plugins/observability/e2e/journeys/single_metric.journey.ts

```diff
@@ -6,10 +6,13 @@
  */
 
 import { journey, step, before } from '@elastic/synthetics';
+import { recordVideo } from '../record_video';
 import { createExploratoryViewUrl } from '../../public/components/shared/exploratory_view/configurations/exploratory_view_url';
 import { loginToKibana, TIMEOUT_60_SEC, waitForLoadingToFinish } from '../utils';
 
 journey('SingleMetric', async ({ page, params }) => {
+  recordVideo(page);
```

Comment:
I would love to avoid calling this repeatedly, but I'm really happy with the new functionality!